benefits/core/migrations/0001_initial.py

```diff
@@ -66,9 +65,7 @@ class Migration(migrations.Migration):
                 ("form_name_placeholder", models.TextField(null=True)),
                 ("form_name_max_length", models.PositiveSmallIntegerField(null=True)),
                 ("unverified_title", models.TextField()),
-                ("unverified_headline", models.TextField()),
                 ("unverified_blurb", models.TextField()),
-                ("eligibility_confirmed_headline", models.TextField()),
```

Comment:
For these changes - I was envisioning we leave the models alone, but consolidate at the PO file and update the references in the data migrations. I'm OK with this approach too, we may end up adding some of this back to be able to change it later.

Reply:
Agreed 👍